### ISSUE TITLE
Error In Server.ts File

### ISSUE THREAD

**Issue by harshdave161** [tberey/ts-node-online-multiplayer-poker#1]
Error in Server.ts file (import privateData from "./config/private.json";) this path is not configured Man !! In github Repo Code There is no Folder like config Where in Can find private.js !!

**Comment by tberey** [comment]
Yeah it’s .gitignored - set up your own, it has secret data in for express sessions

**Comment by tberey** [comment]
See the express-sessions documentation for assistance in that
